src/components/pages/Feedbackscreen.js

```diff
@@ -40,22 +41,13 @@ class Feedbackscreen extends React.Component {
   }
 
   handleSubmit() {
-    fetch("/api/feedback", {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json; charset=UTF-8",
-      },
-      body: JSON.stringify({ message: this.state.value }),
-    })
-      .then((res) => {
-        if (res.status === 200) {
-          this.provideUserFeedback("Feedback erhalten");
-          this.setState({ value: "" });
-        } else {
-          this.provideUserFeedback("Fehler beim Senden");
-        }
+    sendFeedback(this.state.value)
```

Comment:
~Could maybe be renamed to `sendFeedbackByMail`?~
On second thought: it is not important to know here that it is send by mail.